unitOneAssessmentPractice.js

```diff
@@ -4,34 +4,82 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const double = (num) => {
+  return num * 2
+}
+
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+//runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+    let str = num.toString()
```

Comment:
Remember that methods can be chained together `return num.toString().inclues("7")`